src/cascadia/TerminalApp/lib/TerminalAppLib.vcxproj

```diff
@@ -235,7 +239,7 @@
   <ItemDefinitionGroup>
     <ClCompile>
       <PrecompiledHeaderFile>pch.h</PrecompiledHeaderFile>
-      <AdditionalIncludeDirectories>..;$(OpenConsoleDir)\dep\jsoncpp\json;%(AdditionalIncludeDirectories);</AdditionalIncludeDirectories>
+      <AdditionalIncludeDirectories>..;$(OpenConsoleDir)\src\cascadia\WinRTUtils\inc;$(OpenConsoleDir)\dep\jsoncpp\json;%(AdditionalIncludeDirectories);</AdditionalIncludeDirectories>

```

Comment:
Should this particular include project be a part of `cppwinrt.build.pre.props` common?

Reply:
I could go either way on this. I like the idea of having all C++/winrt projects able to access this ;P

Reply:
Decided to put it into cppwinrt pre props (@zadjii-msft may have a feeling about this)

Reply:
I have a feeling, the feeling is 😀